solr/CHANGES.txt

```diff
@@ -47,6 +47,8 @@ Other Changes
 
 * SOLR-14012: Return long value for unique and hll aggregations irrespective of shard count (Munendra S N, hossman)
 
+* SOLR-13492: Disallow explicit GC by default during Solr startup.
```

Comment:
I think this can be moved to 8.6 others section. Also, please include reporter and your name